hints/largest-rectangle-in-histogram.md

```diff
@@ -0,0 +1,39 @@
+<br>
+<details class="hint-accordion">  
+    <summary>Recommended Time & Space Complexity</summary>
+    <p>
+    You should aim for a solution with <code>O(n)</code> time and <code>O(n)</code> space, where <code>n</code> is the size of the input array.
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 1</summary>
+    <p>
+    A rectangle has a length and a breadth. Can you visualize how rectangles are formed in the given input? Considering one bar at a time might help. The height of the bar becomes the length of the rectangle. How can you determine the breadth of the rectangle in which the current bar is present, with the length of the rectangle being the height of the bar? Extending the current bar to the left and right might help determine the rectangle's breadth.
```

Comment:
Why are we using length and breadth?

Don't you think in the context of this problem we should use `width` and `height`. Please read the problem statement again. To be honest, I know it's easy for me to correct this, but I'm honestly a little puzzled that I had to tell you this. 

I'm pretty sure you are better than me at leetcode, so why is your reading and writing skills so subpar? If you are tired, I understand. You can always take a break. I would prefer that, rather than having to give you basic feedback.

Reply:
Sorry, I will try to improve it. I thought people would be comfortable with length and breadth in terms of rectangles. But yeah, I will make sure and be more focused on it now.